quic/s2n-quic-platform/src/message/msg/tests.rs

```diff
@@ -34,6 +34,22 @@ fn test_msghdr<F: FnOnce(&mut msghdr)>(f: F) {
     f(&mut msghdr);
 }
 
+#[cfg(kani)]
+#[allow(dead_code)] // Avoid warning when using stubs.
+mod stubs {
+    use s2n_quic_core::inet::AncillaryData;
+
+    pub fn decode(_msghdr: &libc::msghdr) -> AncillaryData {
+        let mut ancillary_data = AncillaryData::default();
```

Comment:
Note to self: we talked offline about adding the ability to call `any` on a bolero generator. Ideally we would use that here to keep the generators consistent.

https://github.com/camshaft/bolero/issues/194